queries.sql

```diff
@@ -45,7 +45,20 @@ SELECT SUM(escape_attempts) AS escape_attempts, neutered FROM animals GROUP BY n
 SELECT MAX(weight_kg), MIN(weight_kg), species FROM animals GROUP BY species;
 SELECT AVG(escape_attempts) AS avg_escape_attempts, species FROM animals  WHERE date_of_birth BETWEEN '1990/01/01' AND '2020/12/31' GROUP BY species;
 
-
+--What animals belong to Melody Pond?
+SELECT * FROM owners JOIN animals ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT * FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT * FROM Owners FULL OUTER JOIN animals ON owners.id = animals.owner_id;
+-- How many animals are there per species?
+SELECT species.name, COUNT(*) FROM species JOIN  animals ON animals.species_id = species.id GROUP BY species.name;
+-- List all Digimon owned by Jennifer Orwell.
+SELECT * FROM animals JOIN owners ON owners.id = animals.owner_id JOIN species ON species.id = animals.species_id WHERE species.name = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
+-- List all animals owned by Dean Winchester that haven't tried to escape.
+SELECT * FROM animals JOIN owners ON animals.owner_id = owners.id WHERE animals.escape_attempts = 0 AND owners.full_name = 'Dean Winchester';
+--Who owns the most animals?
+SELECT owners.full_name, COUNT(*) FROM animals JOIN owners ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY COUNT DESC LIMIT 1;
```

Comment:
- [ ] You have written incorrectly. It should be 
`SELECT owners.full_name, COUNT(*) FROM animals JOIN owners ON `animals.speceis_id` = owners.id GROUP BY owners.full_name ORDER BY COUNT DESC LIMIT 1;`

Reply:
Thanks @Deepakdanger , it was a sort of misunderstanding.